platformio_core_defs.ini

```diff
@@ -159,7 +159,7 @@ platform_packages =
 
 [core_esp32_2_1_0]
 platform                  = espressif32@2.1.0
-build_flags               = -DESP32_STAGE
+build_flags               = -DESP32_STAGE -Wswitch
```

Comment:
What does this build flag do?

Reply:
Report the `switch` statements that do not implement all options of a `class enum` declaration in a `case`, while that _is_ enabled for ESP8266 (until now I used that to check my switch statements, but it would be helpful to have that for ESP32 too)

Reply:
2 example results: (I've used a couple of these enums in  my code, so wanted to check 😉)
```
.pio/libdeps/display_ESP32_4M316k/ESPeasySerial/ESPEasySerial_ESP32.cpp: In function 'bool pinsChanged(ESPEasySerialPort, int, int)':
.pio/libdeps/display_ESP32_4M316k/ESPeasySerial/ESPEasySerial_ESP32.cpp:33:10: warning: enumeration value 'not_set' not handled in switch [-Wswitch]
   switch (port) {
          ^
```
and
```
src/src/Helpers_tmp/__tmpfile.cpp: In function 'String getPartitionType(uint8_t, uint8_t)':
src/src/Helpers_tmp/__tmpfile.cpp:7499:7: warning: case value '153' not in enumerated type 'esp_partition_subtype_t' [-Wswitch]
       case 0x99: return F("EEPROM"); // Not defined in esp_partition_subtype_t
       ^~~~
```